vis/test/snapshot/README.md

```diff
@@ -0,0 +1,2 @@
+# Snapshot Tests
+Snapshot testing using jest. See https://jestjs.io/docs/en/snapshot-testing for more details
```

Comment:
And the components are rendered with `react-test-renderer`.

Reply:
:+1: